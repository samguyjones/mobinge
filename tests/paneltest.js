import React from 'react';
import Panel from '../src/components/Panel';
import renderer from 'react-test-renderer';

test('Test Render', () => {
  const style = {
    border: '1px solid black'
  };
  var panelRef;
  const component = renderer.create(
    <Panel image="test.gif" style={style} ref={(comp) => {panelRef = comp;}}></Panel>
  );
  let render = component.toJSON();
  expect(render).toMatchSnapshot();
  panelRef.load();
  render = component.toJSON();
  expect(render).toMatchSnapshot();
});
