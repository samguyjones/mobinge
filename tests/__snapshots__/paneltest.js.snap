// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Render 1`] = `
<div
  onDragStart={[Function]}
  style={
    Object {
      "border": "1px solid black",
    }
  }
>
  If you are reading this, you might have lost network connectivity or this app broke.  If you could go to 
  <a
    href="https://www.facebook.com/inhumaneresourcescomic/"
  >
    this link
  </a>
  , click "Send Message" and tell me what you did before you saw this message, I would be grateful.  First, wait a second and see if this disappears and gets replaced by an image.
</div>
`;

exports[`Test Render 2`] = `
<img
  onDragStart={[Function]}
  src="test.gif"
  style={
    Object {
      "border": "1px solid black",
    }
  }
/>
`;
